config/setup.js

```diff
@@ -37,6 +37,32 @@ function signContent(data, callback) {
         });
 }
 
+generateTokenAndStoreInfo = function(username, password){
+  let headers = {}
+  let auth = {
+    username:username,
+    password:password
+  };
+  headers['Authorization'] = basicAuth(auth)
+  headers['Content-Type'] = 'application/json'
+  headers['accept'] = 'application/json'
+  let method = "POST"
+  let url = "https://api.github.com/authorizations"
+  let body = JSON.stringify({
+    "scopes": ["repo", "write:packages", "read:packages"],
+    "note": "For le-git-imate"
```

Comment:
Please use the predefined extension id and update the note as follows:
note": EXTENSION_ID